plugins/vpp/ifplugin/nat_config.go

```diff
@@ -273,6 +315,15 @@ func (plugin *NatConfigurator) DeleteNatGlobalConfig(config *nat.Nat44Global) (e
 		}
 	}
 
+	// Reset virtual reassembly
+	if err := plugin.natHandler.SetDefaultVirtualReassemblyIPv4(); err != nil {
+		return err
+	}
+	if err := plugin.natHandler.SetDefaultVirtualReassemblyIPv6(); err != nil {
+		return err
+	}
+	plugin.log.Debug("Nat virtual reassembly reset for IPv4 & IPv6")
```

Comment:
Let's try to minimize amount of debug logs when not really useful. There another one just below.. more debugs is not always easier during debugging.